article/Experiment/GCF-ablation-study.tex

```diff
@@ -0,0 +1,100 @@
+\subsection{Ablation study for GCF}
```

Comment:
burde man egentlig ikke generelt igennem kalde det aggregator function istedet for at kalde det layer combination.
Selve delen af methoden der står for det kan vist kaldes layer combination.
Men når man snakker om concat, summation eller en vilkårlig anden måde at gøre det på, er det så ikke aggregator funktionen i layer kombination modulet som man snakker om ?

Reply:
Jeg ville nok foretrække med at blive ved med at kalde det layer combination. Layer combination er jo kun metoden man benytter til at combinere de forskellige layers. Det er også det som de andre papers plejer at kalde det.